Makefile

```diff
@@ -29,7 +29,7 @@ clean: ## Remove temporary files generated by the build and tests
 clean-all: clean ## Remove all temporary files plus installed dependencies
 	rm -rf ./node_modules
 
-start: install clean ## Start the development server
+start: build ## Start the development server
```

Comment:
`start` shouldn't rely on `build`. It starts a `webpack-dev-server`, which compiles and serves from memory. It doesn't reference any of the build artifacts.

Reply:
Also, this seems out of scope

Reply:
Yes except the examples are using the build files

Reply:
Ah, I didn't realize `examples/filtering-ticket-groups` was referencing `build/tevomaps.js`. Why not reference `src/index.tsx` instead, as `example/filtering-ticket-groups` is already using webpack?

Reply:
Also, you can run `eamples/using-lib` without running any commands (building or starting).